app/controllers/search_controller.rb

```diff
@@ -29,11 +29,16 @@ def showsearch(title_name)
 
         i=0
         while i < @idresult.size
-            while @idresult['title_results'][i]['id'] != nil
-                @id = @idresult['title_results'][i]['id']
-                @titles = @idresult['title_results'][i]['name']
-                @search = Search.create(search_id: @id, title: @titles)
-                break
+            if @idresult['title_results'] == []
```

Comment:
searches that cause errors:
- "house md": redirects to error page
- "avatar the last airbender": doesn't actually search
- "atla": returns empty results page

Reply:
please fix